.circleci/config.yml

```diff
@@ -22,7 +18,15 @@ jobs:
       TRAVIS_OS_NAME: linux
       ARCH: x86_64
       ARCH_CMD: linux64
-    steps: *ci_steps
+    steps:
```

Comment:
I think it could be simplified to:

```yaml
steps:
  <<: *ci_steps
  - persist_to_workspace:
    root: .
    paths:
      - docs
```

Reply:
I wish, but yaml does not support merging arrays AFAIK.